### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by t11r**
Better integration of OpenSeadragon and icons.

### CODE REVIEW

File: build/create-icons.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
```
Comment (paulpestov) [inline build/create-icons.js:1]: Do we actually have to do this at the postinstall? Can't this be a regular component like `<tify-icon name="test">`?

Comment (t11r) [reply build/create-icons.js:1]: It cannot, unless we want all 6000+ icons to be compiled into the TIFY package; tree shaking does not help. Besides, using icon components like this is exactly what the creators of Vue are doing.

Comment (paulpestov) [reply build/create-icons.js:1]: Hm not even like [this](https://stackoverflow.com/a/61534450)?

Comment (t11r) [reply build/create-icons.js:1]: Nope.

**Review by paulpestov — APPROVED** [review]

**Pull request merged**